examples/oneapi/cpp/BUILD

```diff
@@ -49,10 +49,21 @@ dal_example_suite(
     test_deps = _TEST_DEPS,
 )
 
+dal_example_suite(
```

Comment:
Don't we want to say it's `dal_preview_example_suite`?

Reply:
It makes sense only if the rules are different, currently they are the same for preview/stable functionality. I don't see much sense in having alias for `dal_example_suite`.

Reply:
We have `jaccard` examples in separate rule, just because the example explicitly use TBB functions, so we need dependency on TBB for this particular example.